app/assets/javascripts/components/articles/available_article.jsx

```diff
@@ -60,15 +60,16 @@ export const AvailableArticle = createReactClass({
     const ratingMobileClass = `${ratingClass} tablet-only`;
     const articleLink = <a onClick={this.stop} href={article.url} target="_blank" className="inline">{article.formatted_title}</a>;
 
-    let actionButton;
-    // Show 'Select' button to students
+    let actionSelect;
+    let actionRemove;
     if (this.props.current_user.role === 0) {
```

Comment:
Using current_user.role is not going to be compatible with showing both at once, except for admins.

If a user is enrolled with both the instructor and student roles in the same course, but is not an admin, then they should see both buttons. In that case, the role will probably be 1, but isStudent and isNonstudent should both be true.

Reply:
@ragesoss I got errors using `isStudent` and `isNonStudent` while in specs. That's why I backtracked on this.